e2e/src/pages/scheduleMeetingWidgetPage.ts

```diff
@@ -100,8 +100,9 @@ export class ScheduleMeetingWidgetPage {
   }
 
   async addParticipant(name: string) {
+    await this.participantsCombobox.click();
```

Comment:
Would be good to know why we have to `click` there. If we don't know let's add some comment here.

Reply:
Comment added.